app/images/warning-triangle-grey.svg

```diff
@@ -0,0 +1 @@
+<svg width="18" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M10.143 12.563v-1.697a.264.264 0 0 0-.09-.205.255.255 0 0 0-.196-.09H8.143a.282.282 0 0 0-.206.09.278.278 0 0 0-.08.205v1.697a.31.31 0 0 0 .08.214c.06.053.128.08.206.08h1.714a.267.267 0 0 0 .197-.08c.06-.06.089-.131.089-.214Zm-.018-3.34.16-4.098a.196.196 0 0 0-.089-.17c-.077-.065-.148-.098-.214-.098H8.018c-.066 0-.137.033-.214.098-.06.042-.09.105-.09.188l.152 4.08c0 .06.03.11.09.152.059.036.13.054.214.054H9.82a.363.363 0 0 0 .206-.054c.06-.042.092-.092.098-.152ZM10 .883l6.857 12.572c.209.375.203.75-.018 1.125-.1.173-.24.31-.42.411-.172.101-.36.152-.562.152H2.143a1.132 1.132 0 0 1-.982-.563c-.22-.375-.226-.75-.018-1.125L8 .884a1.14 1.14 0 0 1 1-.598 1.14 1.14 0 0 1 1 .598Z" fill="#BBC0C5"/></svg>
```

Comment:
To ensure we have the correct licensing for this icon and to access the color via css so we can add themeable colors can we use the fontawesome icon instead of adding the svg here.

Reply:
Done